src/api/slack/slack.factory.ts

```diff
@@ -36,7 +39,37 @@ const generateDelayedMergeRequestsMessage = (delayedMrs: IChannelMergeRequests):
     return messages.join('\r');
 };
 
+const generateRankingMessage = (ranking: IRanking, period: string): string => {
```

Comment:
o `period` não poderia estar dentro do IRanking?
Parece ser muito dependente esses 2 valores.

Reply:
não pois o `period` está "acima" do `ranking`. é o período utilizado na pesquisa dos mrs, que são utilizados para montar o ranking. não consegui pensar em outra forma de fazer

<img width="594" alt="image" src="https://user-images.githubusercontent.com/20214425/68549951-f2d5ae80-03dc-11ea-8c7f-2c4d6606c6bb.png">

Reply:
as o IRanking foi montado com base no period, certo?

Reply:
Não vejo mt sentido em deixar separado.
Um ranking sem saber o periodo é meio inútil, poderia criar mais um objeto então, com o IRanking dentro.
Pensa futuramente também, além do period terá o channel

Reply:
ajustei